app/i18n/de/install.php

```diff
@@ -4,7 +4,9 @@
 	'action' => array(
 		'finish' => 'Installation fertigstellen',
 		'fix_errors_before' => 'Bitte Fehler korrigieren, bevor zum nächsten Schritt gesprungen wird.',
+		'keep_install' => 'Keep previous installation',  // TODO: translate
```

Comment:
@thomasE1993 or @ealdraed Any suggestion? Please see also below the 3 more new lines. Thanks :-)